client/components/mma/cancel/cancellationSaves/ContinueMembershipConfirmation.tsx

```diff
@@ -37,19 +43,19 @@ export const ContinueMembershipConfirmation = () => {
 				`}
 			/>
 			<Stack space={4}>
-				<Heading>Thank you for continuing your membership</Heading>
+				<h2 css={headingCss}>Thank you for keeping your Membership</h2>
 				<p>
-					No action is needed from your side. Your membership will
-					continue as normal.
+					The new price of your Membership is £7/month. <br />
+					Your first billing date will be{' '}
+					{cancellationFormatDate(chargeThroughDate)}.
 				</p>
 			</Stack>
-			<div css={[buttonLayoutCss, { textAlign: 'left' }]}>
-				<Button
-					icon={<SvgArrowRightStraight />}
-					iconSide="right"
-					onClick={() => navigate('https://www.theguardian.com')}
-				>
-					Continue reading the Guardian
+			<div css={stackedButtonLeftLayoutCss}>
+				<Button priority="tertiary" onClick={() => navigate('/')}>
+					Back to account overview
```

Comment:
I think the button text should be centred 
`cssOverrides={buttonCentredCss}`

Reply:
Good catch -  I thought I had push this commit 🤦‍♀️